Gems/PhysX/Code/Tests/CharacterGameplayControllerTests.cpp

```diff
@@ -114,41 +117,136 @@ namespace PhysX
+        EXPECT_THAT(basis.m_gameplayController->GetGravityMultiplier(), testing::FloatEq(expectedGravityMultiplier));
     }
 
+    //! Test that verifies that the Gravity Set & Get methods operate as expected while the scene ticks
+    //! and interacts with the entity & its components.
     TEST_F(PhysXDefaultWorldTest, CharacterGameplayController_GravitySetsWhileMoving)
     {
         const float expectedGravityMultiplier = 2.5f;
+        GameplayTestBasis basis(m_testSceneHandle, DefaultGravityMultiplier, DefaultGroundDetectionBoxHeight);
 
-        GameplayTestBasis basis(m_testSceneHandle, DefaultGravityMultiplier, DefaultGroundDetectionBoxHeight, DefaultFloorTransform);
-
-        // Let scene run for a few moments so the entity can be manipulated by gravity from the gameplay component
+        // Let scene run for a few moments so the entity can be manipulated by gravity from the gameplay component.
         int duration = 10;
-
         for (int i = 0; i < duration; i++)
         {
             basis.m_gameplayController->SetGravityMultiplier(expectedGravityMultiplier + i);
             basis.Update();
 
-            EXPECT_THAT(expectedGravityMultiplier + i, testing::FloatEq(basis.m_gameplayController->GetGravityMultiplier()));
+            EXPECT_THAT(basis.m_gameplayController->GetGravityMultiplier(), testing::FloatEq(expectedGravityMultiplier + i));
+        }
+    }
+
+    //! Test that verifies that the Falling Velocity Set & Get methods function as expected.
+    TEST_F(PhysXDefaultWorldTest, CharacterGameplayController_FallingVelocitySets)
+    {
+        const AZ::Vector3 expectedVelocity(0.0f, 0.0f, 22.0f);
+        GameplayTestBasis basis(m_testSceneHandle, DefaultGravityMultiplier, DefaultGroundDetectionBoxHeight);
+
+        // Get the original velocity so we can determine that set actually changes things.
+        auto originalVelocity = basis.m_gameplayController->GetFallingVelocity();
+
+        // Set the falling velocity to the expected + original velocity to verify that the velociy will change.
+        basis.m_gameplayController->SetFallingVelocity(originalVelocity + expectedVelocity);
+
+        // Get the end velocity and validate that it is the expected value.
+        auto endVelocity = basis.m_gameplayController->GetFallingVelocity();
+        EXPECT_THAT(endVelocity.GetZ(), testing::FloatNear(expectedVelocity.GetZ(), 0.001f));
+    }
+
+    //! Test the verifies that the Falling Velocity sets and uses the set value to interact with the tick incrementing.
+    TEST_F(PhysXDefaultWorldTest, CharacterGameplayController_FallingVelocitySetsWhileMoving)
+    {
+        const AZ::Vector3 expectedVelocity(0.0f, 0.0f, 22.0f);
+        GameplayTestBasis basis(m_testSceneHandle, DefaultGravityMultiplier, DefaultGroundDetectionBoxHeight);
+
+        // Get the original velocity so we can determine that set actually changes things.
+        auto originalVelocity = basis.m_gameplayController->GetFallingVelocity();
```

Comment:
Again the original velocity is going to be zero because you haven't ticked physics yet. So adding it to things is not going to do anything, and doesn't really make any sense.

Reply:
Removed